plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -319,6 +328,18 @@ public static IcebergFileFormat getFileFormat(Map<String, String> storagePropert
                 .toUpperCase(Locale.ENGLISH)));
     }
 
+    public static IcebergMetadataCompressionCodec getMetadataCompressionCodec(Table table)
+    {
+        return getMetadataCompressionCodec(table.properties());
+    }
+
+    public static IcebergMetadataCompressionCodec getMetadataCompressionCodec(Map<String, String> storageProperties)
+    {
+        return IcebergMetadataCompressionCodec.fromIceberg(Codec.fromName(storageProperties
+                .getOrDefault(METADATA_COMPRESSION_CODEC, METADATA_COMPRESSION_DEFAULT)
```

Comment:
Use 

```
.getOrDefault("write.metadata.compression-codec", METADATA_COMPRESSION_DEFAULT)
```

The storage properties are Iceberg specific, not Trino specific.

Reply:
Let's modify also `io.trino.plugin.iceberg.BaseIcebergMaterializedViewTest#testShowCreate` to contain a `gzip` metadata compression codec.

optional: We could also add a test for iceberg tables showing the two approaches (when metadata codec is none it doesn't show up in show create, when it is `gzip` it shows up in show create).

Reply:
> The storage properties are Iceberg specific, not Trino specific.

Thanks for catching that. It means these constants naming might be little confusing. 

> Let's modify also io.trino.plugin.iceberg.BaseIcebergMaterializedViewTest#testShowCreate to contain a gzip metadata compression codec.

Updated `BaseIcebergConnectorTest#testMetadataCompressionCodecGzip` instead of `BaseIcebergMaterializedViewTest`. 

> We could also add a test for iceberg tables showing the two approaches (when metadata codec is none it doesn't show up in show create, when it is gzip it shows up in show create).

Added `SHOW CREATE TABLE ` check for `gzip` case. Regarding `none` case, it's covered in existing tests `BaseIcebergConnectorTest#testShowCreateTable`.